### PR TITLE
Make ci green

### PR DESCRIPTION
**Pull request by enricosada**
Revert to a version of .net core sdk installed in azure pipeline agents

### CODE REVIEW

File: build.proj [hunk 1]
```diff
@@ -19,7 +19,8 @@
   </Target>
 
   <Target Name="VSTest">
-    <Exec Command="dotnet test Equinox.sln $(Cfg) $(TestOptions)" />
+    <Exec Command="dotnet test tests/Equinox.MemoryStore.Integration $(Cfg) $(TestOptions)" />
```
Comment (bartelink) [inline build.proj]: what's the problem with the other approach ? the cosmos branch will bring more test assemblies and I'd ideally like to have them be picked up automatically ?

**Comment by bartelink** [comment]
<nudge> to make things truly green across the board, there's a single failing test on MacOs and Linux (wild guess says it's some unbox that needs a type hint or something)

File: build.proj (hunk 1 above)
Comment (bartelink) [reply build.proj]: WIP testlist from cosmos (not yet pushed)

```
  <Target Name="VSTest">
    <Exec Command="dotnet test tests/Equinox.Cosmos.Integration $(Cfg) $(TestOptions)" />
    <Exec Command="dotnet test tests/Equinox.EventStore.Integration $(Cfg) $(TestOptions)" />
    <Exec Command="dotnet test tests/Equinox.MemoryStore.Integration $(Cfg) $(TestOptions)" />
    <Exec Command="dotnet test samples/Store/Integration $(Cfg) $(TestOptions)" />
    <Exec Command="dotnet test samples/Store/Domain.Tests $(Cfg) $(TestOptions)" />
  </Target>
```
ideally the above minus the cosmos one would go in this PR

**Comment by bartelink** [comment]
(happy to see this squash merged when other 2 tests suites are re-included in the test run)

File: build.proj (hunk 1 above)
Comment (michaelliao5) [reply build.proj]: i think the issue is that dotnet test *.sln doesnt work on osx/unix, need to specify the fsproj directly (not the sln)
ln has some of quirks in os/linux according to what Enrico told me

Comment (bartelink) [reply build.proj]: @michaelliao5 yes, see below - I read too fast ! But my point (is there an issue about them fixing that somewhere?) remains (for example there are 2 other test assemblies (listed above) that the sln used to pick up)

**Comment by enricosada** [comment]
@ruben rebased, squashed and updated the commit message.

it disable some things, who can be reenabled in successive PR if we get the fix, but ci should be green as possibile

**Comment by bartelink** [comment]
Beautiful 💟 (BTW was talking of doiung a squash merge - have no problem with the commits being separate on the PR branch)

**Comment by bartelink** [comment]
still doing an ugly squash merge to get a link back to this into the message ;)

**Pull request merged**